docs/docs/Get-Started/get-started-installation.md

```diff
@@ -6,166 +6,228 @@ slug: /get-started-installation
+    1. Navigate to where you want your virtual environment to be created, and create it with `uv`.
+Replace `VENV_NAME` with your preferred name for your virtual environment.
+```
+uv venv VENV_NAME
+```
+2. Start the virtual environment.
+```
+source VENV_NAME/bin/activate
 ```
+Your shell's prompt changes to display that you're currently working in a virtual environment.
+```
+(VENV_NAME) ➜  langflow git:(main) ✗
+```
+3. To deactivate the virtual environment and return to your regular shell, type `deactivate`.
+To delete the virtual environment, type `rm -rf VENV_NAME`.
+  </TabItem>
+  <TabItem value="Windows" label="Windows">
+1. Navigate to where you want your virtual environment to be created, and create it with `uv`.
+Replace `VENV_NAME` with your preferred name for your virtual environment.
+```
+uv venv VENV_NAME
+```
+2. Start the virtual environment.
+```shell
+VENV_NAME\Scripts\activate
+```
+Your shell's prompt changes to display that you're currently working in a virtual environment.
+```
+(VENV_NAME) PS C:/users/username/langflow-dir>
+```
+3. To deactivate the virtual environment and return to your regular shell, type `deactivate`.
+To delete the virtual environment, type `Remove-Item VENV_NAME`.
+  </TabItem>
+  </Tabs>
+  <TabItem value="Linux" label="Linux">
+    1. Navigate to where you want your virtual environment to be created, and create it with `uv`.
+Replace `VENV_NAME` with your preferred name for your virtual environment.
+```
+uv venv VENV_NAME
+```
+2. Start the virtual environment.
+```
+source VENV_NAME/bin/activate
+```
+Your shell's prompt changes to display that you're currently working in a virtual environment.
+```
+(VENV_NAME) ➜  langflow git:(main) ✗
+```
+3. To deactivate the virtual environment and return to your regular shell, type `deactivate`.
+To delete the virtual environment, type `rm -rf VENV_NAME`.
+  </TabItem>
+</details>
 
-</TabItem>
-<TabItem value="pip" label="pip">
 
-```bash
-pip install langflow==1.3.2
-```
+<Tabs groupId="os">
+  <TabItem value="macOS" label="macOS">
 
-</TabItem>
-</Tabs>
+    1. To install Langflow, run the following command.
+    ```bash
+    uv pip install langflow
+    ```
 
-To reinstall Langflow and all of its dependencies, add the `--force-reinstall` flag to the command.
-<Tabs groupId="package-manager">
-<TabItem value="uv" label="uv" default>
+  </TabItem>
+  <TabItem value="Windows" label="Windows">
+  :::important
+  Windows installations of Langflow OSS require a C++ compiler, such as [Visual Studio](https://visualstudio.microsoft.com/downloads/), that may not be present on your system. If you receive a `C++ Build Tools Required!` error, follow the on-screen prompt to install Microsoft C++ Build Tools, or visit the Visual Studio download link above.
+  :::
 
-```bash
-uv pip install langflow --force-reinstall
-```
+      1. To install Langflow, run the following command.
+      ```bash
+      uv pip install langflow
+      ```
+
+  :::tip
```

Comment:
Having this tip admonition so close to the important admonition above makes the screen look a little cluttered. Suggest removing the tip and presenting the information as regular text instead.